### PR TITLE
Logo fixed for the sidebar

### PR DESCRIPTION
**Pull request by qandeelarshad**
The logo image was cropped and added to make a better fit on the sidebar.

### CODE REVIEW

**Review by AbdulSir — CHANGES_REQUESTED** [review]
The sidebar logo looks great! However, I noticed that the mobile logo layout takes up half of the viewport (screen). Is that something you're planning to fix in the branch?

Screenshot:
image

File: frontend/src/containers/TheHeader.vue [hunk 1]
```diff
@@ -11,7 +11,7 @@
       @click="$store.commit('toggleSidebarDesktop')"
     />
     <CHeaderBrand class="mx-auto d-lg-none" to="/">
-      <img src="../assets/polaris.svg" class="img-fluid" alt="Responsive image">
+      <img src="../assets/autotracks_blue_crp.png" class="img-fluid" alt="Responsive image">
```
Comment (purplesmurf1998) [inline frontend/src/containers/TheHeader.vue:14]: Have you tried putting a fixed height style for the image? Or maybe a max height. Here is a potential fix:
`Responsive image`

and in the style tags
```
<styles scoped>
.header-img {
     max-height: 100px;
}
</styles>
```

**Review by purplesmurf1998 — CHANGES_REQUESTED** [review]

File: frontend/src/containers/TheHeader.vue (hunk 1 above)
Comment (Emperor42) [reply frontend/src/containers/TheHeader.vue]: Requested change has now been committed

File: frontend/src/containers/TheHeader.vue [hunk 2]
```diff
@@ -11,7 +17,7 @@
       @click="$store.commit('toggleSidebarDesktop')"
     />
     <CHeaderBrand class="mx-auto d-lg-none" to="/">
-      <img src="../assets/polaris.svg" class="img-fluid" alt="Responsive image">
+      <img src="../assets/autotracks_blue_crp.png" class="img-fluid header-img" alt="Responsive image">
```
Comment (Emperor42) [inline frontend/src/containers/TheHeader.vue:20]: Change on file completed as requested

**Review by Emperor42 — COMMENTED** [review]
Logo look great, change to set a maximum size for the header has been added

**Review by AbdulSir — APPROVED** [review]
The icons look great on desktop and mobile. This PR is ready to be merged.

**Review by purplesmurf1998 — APPROVED** [review]
Logo looks good

**Pull request merged**